ush/test/config/bufr_bufr4backend_atms.yaml

```diff
@@ -0,0 +1,58 @@
+time window:
+  begin: "2018-04-14T21:00:00Z"
+  end: "2023-12-15T03:00:00Z"
+
```

Comment:
Do these yamls need to be bound inclusive?

Reply:
Added.

Reply:
added